src/Parser/MistweaverMonk/CombatLogParser.js

```diff
@@ -661,6 +661,89 @@ class CombatLogParser extends MainCombatLogParser {
           <LowHealthHealingTab parser={this} />
         ),
       },
+      {
+        title: 'Player Log Data',
+        url: 'player-log-data',
+        render: () => (
+          <div>
```

Comment:
Please extract this into a new component, it's too long and easy to separate to keep in the parser. You can always pass `this` as property to the new component like is done for the `LowHealthHealingTab`.